hack/generator/pkg/astmodel/comments.go

```diff
@@ -42,8 +42,24 @@ func addDocComment(commentList *[]*ast.Comment, comment string, width int) {
 	}
 }
 
-// formatDocComment splits the supplied comment string up ready for use as a documentation comment
-func formatDocComment(comment string, width int) []string {
+func addComment(commentList *[]*ast.Comment, comment string) {
+	line := strings.TrimSpace(comment)
```

Comment:
Would it make sense for addWrappedComment to call addComment, rather than duplicating the body of it in the loop?

Reply:
Good idea; I'll reduce duplication by adding reuse.